crates/store/re_log_types/src/path/entity_path_filter.rs

```diff
@@ -89,6 +89,9 @@ impl TryFrom<&str> for EntityPathFilter {
 /// The last rule matching `/world/car/hood` is `- /world/car/**`, so it is excluded.
 /// The last rule matching `/world` is `- /world`, so it is excluded.
 /// The last rule matching `/world/house` is `+ /world/**`, so it is included.
+///
+/// Unless otherwise specified, the [`ResolvedEntityPathFilter`] will filter out entities
+/// at [`EntityPath::partition_properties`].
```

Comment:
That's not really a property of the `ResolvedEntityPathFilter` itself: if I were to create one without going through `EntityPathFilter` first (not possible right now iirc) then it would *not*. filter out `partition_properties`
I.e. it instead a property of `EntityPathFilter` to _produced_ such `ResolvedEntityPathFilter`s that it will filter out `partition_properties` if not specified otherwise